app/views/needs/_form_fields.html.haml

```diff
@@ -25,16 +31,11 @@
     = f.label :age_range_ids do
       Age Range of Children
       = f.select :age_range_ids, options_for_select(AgeRange.all.each.with_object({}) { |a, o| o["#{a.min}-#{a.max}"] = a.id }, @need.age_range_ids), { include_blank: true }, {class: 'multiple', multiple: true, name: 'need[age_range_ids][]'}
-.grid-x
-  .cell
-    = f.label :office_id do
-      Office
-      = f.select :office_id, options_for_select(policy_scope(Office).map { |o| [o.name, o.id]}), { include_blank: false }, class: 'single'
 .grid-x
   .cell
     = f.label :race do
       Preferred Culture
-      = f.collection_select :race, Race.all, :id, :name, class: 'single', include_blank: true
+      = f.select :race, options_for_select(Race.all.map { |r| [r.name, r.id]}), { include_blank: true }, class: 'single'
```

Comment:
Could add a method to `NeedsHelper` to get these options,

e.g. 

``` 
def race_options
  Race.pluck(:name, :id)
end
```

And then in view:
```
= f.select :race, options_for_select(race_options), { include_blank: true }, class: 'single'
```